src/pages/accomodation/Accommodation.tsx

```diff
@@ -20,14 +23,27 @@ function Accommodation() {
 
   const { isPending, data, isError } = useGetSingleAccommodationQuery(accommodationId as string);
 
+  const { mutateAsync } = useBookingRoom();
+
+  const submitReservation = async (reservationData: {
+    startDate: string;
+    endDate: string;
+    accommodationId: string;
+  }): Promise<void> => {
+    try {
```

Comment:
remove try, catch, Error comes two times, one from catch, one from axios interceptor

Reply:
is there the toast notification implemented already into interceptor?

Reply:
yes toast notification is already  implemented i will delete try/catch